optaweb-employee-rostering-benchmark/src/main/java/org/optaweb/employeerostering/OptaWebEmployeeRosteringBenchmarkApplication.java

```diff
@@ -53,7 +53,8 @@
     private final Logger logger = LoggerFactory.getLogger(getClass());
 
     public static void main(String[] args) {
-        Quarkus.run(args);
+        Quarkus.run(OptaWebEmployeeRosteringBenchmarkApplication.class,
+                args);
```

Comment:
Your IDE keeps doing this. :-) These should be on one line.

Reply:
Na, it me trying to keep it to 80 so it fits my screen

Reply:
Are you using a terminal from 1981?
We agreed on line length of 120.

Reply:
No, and IDE with large font which make it effectively 80; I can change the line

Reply:
Ah, I see. 
If the default length is causing issues, perhaps we should discuss that as a team? 
Feel free to bring it up,  no point in enforcing something which may be needlessly painful.